www/android/geolocation.js

```diff
@@ -38,13 +39,14 @@ module.exports = {
                 error(new PositionError(PositionError.PERMISSION_DENIED, 'Illegal Access'));
             }
         };
-        exec(win, fail, 'Geolocation', 'getPermission', []);
+        exec(win, fail, 'Geolocation', 'getPermission', [args.enableHighAccuracy]);
```

Comment:
here `args` is optional and may be undefined. Therefore we need to check `args` first.

We also need to assert that `enableHighAccuracy` is a boolean, otherwise we will get an obscure `Invalid Access` error (actually a JSON error thrown by [getBoolean](https://github.com/apache/cordova-plugin-geolocation/pull/250/files#diff-120ceef27c4964f952ea0ecbb5786d27a15351c1e234328794c6e7eae4eb9c66R51). This is also triggered if `enableHighAccuracy` is left undefined, so we will need to populate this with a default value.